extensions/query-store/package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "query-store",
+  "displayName": "%queryStore.displayName%",
+  "description": "%queryStore.description%",
+  "version": "0.1.0",
+  "publisher": "Microsoft",
+  "preview": true,
+  "engines": {
+    "vscode": "*",
+    "azdata": ">=1.45.0"
+  },
+  "license": "https://raw.githubusercontent.com/Microsoft/azuredatastudio/main/LICENSE.txt",
+  "icon": "images/extension.png",
+  "aiKey": "29a207bb14f84905966a8f22524cb730-25407f35-11b6-4d4e-8114-ab9e843cb52f-7380",
+  "activationEvents": [
+    "*"
```

Comment:
How about adding a tracking issue for following up on this? I imagine it's too early to know what you want this to be to start with, so that'd be a good way to make sure we don't forget?

Reply:
created https://github.com/microsoft/azuredatastudio/issues/23269 for tracking